### ISSUE TITLE
Add dropout and batchnorm to classic CNN baseline to avoid overfitting

### ISSUE THREAD

**Issue by alzaia** [mila-aia/image_classification_simulation#32]
This baseline needs dropout and BN layers to avoid overfitting (based on training runs done so far on the Office31 dataset).
